semcore/data-table/src/Body.tsx

```diff
@@ -58,12 +59,59 @@ class Body extends Component<AsProps, State> {
   firstRowRef = React.createRef<HTMLDivElement>();
   firstRowResizeObserver: ResizeObserver | null = null;
 
+  lockedCell: [HTMLElement | null, boolean] = [null, false];
+
   getRowHeight = () => {
     const { virtualScroll } = this.asProps;
     const rowHeightFromProps = typeof virtualScroll === 'object' && virtualScroll?.rowHeight;
     return rowHeightFromProps || this.state.rowHeight;
   };
 
+  handleKeyDown = (e: React.KeyboardEvent) => {
```

Comment:
Not sure where to put it, but scroll when navigation with arorws feels a bit random. When you navigate verticall it scrolls exactly the cell width, but when scrolls horiztonally it scrolls only for a part of cell, so you can read a cell outside of scroll viewport so it's not happening smoothly.

Reply:
I've added `preventDefault` to the keyDown handler in the Table root